lua/wikis/commons/Infobox/Map.lua

```diff
@@ -109,6 +133,8 @@ function Map:_setLpdbData(args)
 			return 'creator' .. index, value
 		end), {
 			creator = self.creators[1],
```

Comment:
atm we store creator, creator1, creator2, ...
from what i see via fo's tool page there is only usage in very few places, so porting to
creator1, creator2, ... seems reasonable to me
i will prep on wiki consumers later today

Reply:
yeah changing it is exactly what i meant^^

Reply:
consumers should all be adjusted